packages/ui/overlay/header/overlayHeader.css

```diff
@@ -12,6 +12,8 @@
 
 .headerInfo {
   display: flex;
+
+  max-width: 95%;
```

Comment:
where are allocated the rest of 5%?

Reply:
They would go to the `Close` button. But it's not a reliable method.
I would recommend we go with a fixed min-width:
https://github.com/figshare/fcl/pull/113